docs/upgrades/unreleased.md

```diff
@@ -41,6 +41,18 @@
             configText += `config.${prop} = process.env.${constantCase(prop)} || "${config[prop]?.toString().trim()}";\n`;
         ```
 
+* Update import in _src/templates/angular/server.bundle.ts_
```

Comment:
Customer will need to update Angular and Typescript as well
Can you add these steps as well?

Reply:
Updated